### ISSUE TITLE
HA Add-on file upload: "Maximum request body size 16777216 exceeded"

### ISSUE THREAD

**Issue by jens-maus** [jens-maus/RaspberryMatic#1294]
**Describe the bug**
When trying to upload either a backup or CCU add-on package that is larger than 16MB in total size the HomeAssistant Add-on of RaspberryMatic does not proceed but brings up a `413 Request Entity is too large` error as shown here:

<img width="592" alt="Bildschirmfoto 2021-06-10 um 11 38 40" src="https://user-images.githubusercontent.com/5887769/121502383-6b00bd80-c9e0-11eb-8bd3-522f20257759.png">

In addition, the response of the HTTP request to upload these files via `fileupload.ccc` coming from ingress looks like:

```
Maximum request body size 16777216 exceeded, actual body size 16778032
```

Furthermore, if trying to directly upload these files outside of the HA ingress routing (by using e.g. port `8088` after having set this to allow to directly access the `lighttpd` webserver within the add-on) perfectly works and does not bring up the mentioned error.

**Steps to reproduce the behavior**
1. Try to upload a file lower than 16MB in size via the ingress routed RaspberryMatic add-on webpage
2. Notice that the upload works and no error is displayed
3. Try to upload a file > 16MB in size via ingress routed webpage
4. Notice that the upload fails and that the browser console brings up the mentioned error
5. Retry the same procedures but by directly using the lighttpd webserver (e.g. port 8088) after having enabled that port in the Add-on settings.
6. Notice that the upload now proceeds correctly for files > 16MB in size.

**Expected behavior**
Even via HA ingress routing file uploads should be able to succeed for file > 16MB in size.

**Screenshots**
see above

**System information:**
 - RaspberryMatic Version: 3.57.5.20210525
 - Used Hardware: ova
 - Used HomeMatic RF-Module: n/a

**Additional context**
This problem seems to be an upstream issue in the HomeAssistant webserver or ingress setup not allowing content to be sent via a POST request > 16MB in size.

**Comment by jens-maus** [comment]
Please note, this issue should be now fixed starting with Home Assistant core-2021.7.0 and Supervisor 2021.06.7

**Comment by jens-maus** [comment]
Sorry for reopening this issue again, but it seems (cf. https://github.com/home-assistant/supervisor/issues/2950) this issue is not fully solved by the HA devs yet (at least not in a way our addon can utilize it). So for the time being I would like to keep this issue open.
